apps/aehttp/priv/swagger.json

```diff
@@ -409,10 +407,8 @@
               "type" : "object",
               "properties" : {
                 "count" : {
-                  "type" : "integer",
-                  "format" : "int64",
                   "description" : "Count",
-                  "minimum" : 1
+                  "$ref" : "#/definitions/UInt32"
```

Comment:
I see this now allows the value of 0. OK

Reply:
We allow block height to be -1 even though we can't have blocks at height 0 even 😛

Reply:
This is a special block height

Reply:
No, it is a hack...